src/UnisonShare/Route.elm

```diff
@@ -0,0 +1,312 @@
+module UnisonShare.Route exposing
+    ( Route(..)
+    , fromUrl
+    , navigate
+    , navigateToCurrentPerspective
+    , navigateToDefinition
+    , navigateToLatestCodebase
+    , navigateToPerspective
+    , perspectiveParams
+    , replacePerspective
+    , toDefinition
+    , toRoute
+    , toUrlString
+    , updatePerspectiveParams
+    )
```

Comment:
This is pretty much the old route, now duplicated between UnisonLocal and UnisonShare. The main difference (for right now) is that the `UnisonShare.Route` has a `Catalog` variant